content/applications/finance/accounting/fiscal_localizations/localizations/mexico.rst

```diff
@@ -118,6 +118,11 @@ click on *Update information* under your company name.
 In the resulting form, put your full address (including zip code), RFC (VAT number), and the rest of
 the data.
 
+.. important::
+   From a legal point of view, a Mexican company must use the local currency (MXN). Therefore,
+   Odoo does not provide features to manage an alternative configuration. If you want
+   to manage another currency, let MXN be the default currency and use price list instead.
+
```

Comment:
Thanks for the change @vbe-odoo 

It seems you forgot to apply the two other points in my previous review:
- 100th character line break (see [RST guidelines](https://www.odoo.com/documentation/14.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))
- write "and use **a pricelist** instead" instead of "and use price list instead"

Additional suggestion:
- "pricelist" could reference the appropriate doc (see in the suggestion below)

```suggestion
.. important::
   From a legal point of view, a Mexican company must use the local currency (MXN). Therefore, Odoo
   does not provide features to manage an alternative configuration. If you want to manage another
   currency, let MXN be the default currency and use a :doc:`pricelist
   </applications/sales/sales/products_prices/prices/pricing>` instead.

```

Thanks :)

Reply:
@jcs-odoo Oh sorry! my bad! I will commit your suggestion!
And good idea to add the link to the functional DOC of payment terms 💯 
cc' @ren-odoo